syncr_backend/bin/run_backend.py

```diff
@@ -10,7 +10,9 @@
 from syncr_backend.init import drop_init
 from syncr_backend.init import node_init
 from syncr_backend.metadata.drop_metadata import send_my_pub_key
-from syncr_backend.network.listen_requests import listen_requests
+from syncr_backend.network.handle_frontend import handle_request as \
+    handle_frontend_request
```

Comment:
There already is a function called handle_frontend_request in handle_frontend. This should be imported as a different name.